modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4026,6 +4026,7 @@ confirm=Confirm
 confirm-folder-name=Confirm Folder Name
 confirm-object-definition-name=Confirm Object Definition Name
 confirm-password=Confirm Password
+confirm-publishing=Confirm publishing
```

Comment:
@andremhirai Please, update it to `Confirm Publishing`. This language key represents a title and BChan likes to follow the rules of https://capitalizemytitle.com/